### ISSUE TITLE
Error using provider version 0.6.1 with terraform cloud

### ISSUE THREAD

**Issue by AMKamel** [Backblaze/terraform-provider-b2#26]
When I run a resource with this version inside terraform cloud, I got an error 
```
│ Error: [1568] Error loading Python lib '/tmp/_MEIGPtCL8/libpython3.9.so.1.0': dlopen: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.29' not found (required by /tmp/_MEIGPtCL8/libpython3.9.so.1.0)
```
I have to downgrade to `0.6.0` to be able to use terraform cloud
https://i.imgur.com/f0Pru4f.png

**Comment by Hackworth** [comment]
Same issue here.

**Comment by mlech-reef** [comment]
We have a similar issue in B2 CLI (https://github.com/Backblaze/B2_Command_Line_Tool/issues/750) when the CLI bundle is compiled in Github Actions on `ubuntu-latest`. See my comment with probable cause: https://github.com/Backblaze/B2_Command_Line_Tool/issues/750#issuecomment-925142750

**Comment by mlech-reef** [comment]
Related to #25

**Comment by mszumocki-reef** [comment]
Fix will go out with 0.7 release.
